### ISSUE TITLE
Compiling with toES5:false does not compile correctly

### ISSUE THREAD

**Issue by terotests** [terotests/ASTWalker#6]
Arrow functions are not compiled correcltly, it seems.

**Comment by LukasOchmann** [comment]
i dont see what is wrong. tested 0.95 with AST of these:

1. `() => 1`
2. `() => console.log('hello world')`
3. `(t) => t`
4. `(t) => console.log(t)`

using `ASTWalker({
        defaultNamespace: "DOM",
        toES5: true,
    });`it created:
1. `function(){ return 1}.bind(this);`
2. `function() {
  console.log("hello world");
}.bind(this);`
3. `function(t){ return t}.bind(this);`
4. `function(t) {
  console.log(t);
}.bind(this);`

**Comment by terotests** [comment]
Hi, sorry - maybe I was unclear - the problem comes if it set to `false` - so yes, those examples do work, but if `toES5: false` - then I had some problems (at least in the example I was testing with). But I will fix that when have time (hopefully this week)

**Comment by LukasOchmann** [comment]
ohhh yaeh i see it now. 1. and 3. of my tests are failing with toES5:false.
or should they stay as arrow functions?

**Comment by terotests** [comment]
Yep - there is some, possibly quite easy to fix bug, when generating the Arrow functions
